misk-feature/src/main/kotlin/misk/feature/FeatureFlags.kt

```diff
@@ -70,5 +70,6 @@ data class Attributes(
   // NB: LaunchDarkly uses typed Gson attributes. We could leak that through, but that could make
   // code unwieldly. Numerical attributes are likely to be rarely used, so we make it a separate,
   // optional field rather than trying to account for multiple possible attribute types.
-  val number: Map<String, Number>? = null
+  val number: Map<String, Number>? = null,
+  val anonymous: Boolean = false
```

Comment:
I think this field would benefit from a little documentation. It is not clear that anonymous means that there are no attributes to be associated.